dashboard/test/helpers/survey_results_helper_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class SurveyResultsHelperTest < ActionView::TestCase
+  include SurveyResultsHelper
+
+  def sign_in(user)
+    # override the default sign_in helper because we don't actually have a request or anything here
+    stubs(:current_user).returns user
+  end
+
+  setup do
+    @teacher = create(:teacher, created_at: 3.weeks.ago)
+    sign_in @teacher
+  end
+
+  test 'teacher account existed 14 days' do
+    assert account_existed_14_days?
+  end
+
+  test 'teacher account newer than 14 days' do
+    @teacher.update(created_at: 1.week.ago)
```

Comment:
nit: Use `@teacher.update!` to facilitate debugging if per chance the update fails.